Nukeeper.BitBucketLocal/BitBucketLocalPlatform.cs

```diff
@@ -29,8 +30,9 @@ public void Initialise(AuthSettings settings)
             _settings = settings;
             var httpClient = new HttpClient
             {
-                BaseAddress = settings.ApiBase
-            };
+                BaseAddress = new Uri($"{settings.ApiBase.Scheme}://{settings.ApiBase.Authority}")
```

Comment:
Isn't this exactly the same in the end as the old line?

Reply:
No... this removes any path in the URL